docs/UserGuide.md

```diff
@@ -40,6 +40,48 @@ Commands in this user guide follow this format:
 * Extraneous parameters for commands that do not take in parameters (such as `help`, `ilist`, `exit` and `clear`) will be ignored.<br>
   e.g. if the command specifies `help 123`, it will be interpreted as `help`.
 
+<div markdown="block" class="alert alert-primary">
+**:information_source: Command parsing**<br>
+<div markdown="block" class="alert alert-info">
+All commands in SunRez default to taking the latest occurrence of a duplicate parameter **except [tags](#tag)**. 
+For example, if you key in this command:
+
+`radd n/John Doe n/Timmy Tan p/91234567 e/e0123456@u.nus.edu y/3`
+
+The resident's name will take the **later** occurrence of name and create a resident named "Timmy Tan".
+
+On the other hand, [Tags](#tag) will collate ALL tags specified in a command.
```

Comment:
Hmm, what do you mean by "Tags will collate ALL tags..."?

Reply:
I wanted to specify that` t/one t/two t/three` will create a list of tags as `{one, two, three}` as opposed to taking the latest occurrence. Do you have an idea on how to improve the phrasing?

Reply:
Maybe something like "On the other hand, all tag parameters specified will be taken. For example, ..."

Reply:
Updated.